test/test_tk205.py

```diff
@@ -89,6 +92,17 @@ def test_xlsx_template_creation():
     for rs in rss:
         tk205.template(rs,'build/templates')
 
+def test_json_yaml_translation():
+    example_dir = 'build/examples/json'
+    for example in os.listdir(example_dir):
+        if '~$' not in example:
```

Comment:
Why do you need this check here?

Reply:
It appears I do not need to check this. It is now removed.